### PR TITLE
dash/docs-es-modules

### PR DESCRIPTION
**Pull request by sebastianbochan**
Added dashboard es-modules docs.

### CODE REVIEW

**Review by goransle — APPROVED** [review]
LGTM

File: docs/dashboards/installation-with-es-modules.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            }
+        ]
+    });
+```
+
+## To load a plugin
+A plugin is a third party/community made Highcharts Dashboards addon.
+First, make sure that a plugin support loading over NPM and load the required
+files. In example `DataGrid`` supports NPM loading, so after installing the
+package you could initialize it like:
```
Comment (pawellysy) [inline docs/dashboards/installation-with-es-modules.md:102]:
```suggestion
First, make sure that a plugin supports loading over NPM and load the required
files. In the example `DataGrid` supports NPM loading, so after installing the
package you could initialise it like this:
```

File: docs/dashboards/installation-with-es-modules.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        });
+
+    </script>
+```
+
+## Creating a custom bundle (ES6 module)
+
+The advantage of core files over packages is, that only the required features
+are loaded. This reduces the total download size. We can create a bundle of all
+files to improve the load size and load time further. Create a NodeJS project
```
Comment (pawellysy) [inline docs/dashboards/installation-with-es-modules.md:57]:
```suggestion
files to further improve the load size and load time. Create a NodeJS project
```

**Review by pawellysy — CHANGES_REQUESTED** [review]
Looks good! Just small typos fixes

File: docs/dashboards/installation-with-es-modules.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+```js
+    import Dashboards from 'https://code.highcharts.com/dashboards/es-modules/masters/dashboards.src.js';
+
+    Dashboards.board('container', {
+        editMode: {
+            enabled: true,
+            contextMenu: {
+                enabled: true,
+                items: ['editMode']
```
Comment (karolkolodziej) [inline docs/dashboards/installation-with-es-modules.md:70]: We don't need that: `items: ['editMode']`

**Review by karolkolodziej — APPROVED** [review]

File: docs/dashboards/installation-with-es-modules.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+## Creating a custom bundle (ES6 module)
+
+The advantage of core files over packages is, that only the required features
+are loaded. This reduces the total download size. We can create a bundle of all
+files to further improve the load size and load time. Create a NodeJS project
+and install Highcharts and Webpack as NPM packages.
+
+For a line chart create the JavaScript files as shown below.
```
Comment (pawellysy) [inline docs/dashboards/installation-with-es-modules.md:59]: Not `line chart`

**Review by pawellysy — CHANGES_REQUESTED** [review]

**Review by pawellysy — APPROVED** [review]

**Pull request merged**